src/settings_menu.cc

```diff
@@ -66,7 +65,7 @@ namespace roadrun
       case KEY_DOWN:
         if(highlight == 2)
```

Comment:
perhaps create an enum instead of just `2`?

Reply:
I think this was a quick hack because the array we originally tried to use wasn't working. This issue is now noted in #47, and won't be resolved in this PR.